packaging/installer/kickstart-ng.sh

```diff
@@ -0,0 +1,842 @@
+#!/bin/sh
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# ======================================================================
+# Constants
+
+REPOCONFIG_URL_PREFIX="https://packagecloud.io/netdata/netdata-repoconfig/packages"
+REPOCONFIG_VERSION="1-1"
+PATH="${PATH}:/usr/local/bin:/usr/local/sbin"
+
+# ======================================================================
+# Defaults for environment variables
+
+RELEASE_CHANNEL="nightly"
+NETDATA_CLAIM_URL="https://app.netdata.cloud"
+NETDATA_USER_CONFIG_DIR="/etc/netdata"
+NETDATA_AUTO_UPDATES="1"
+NETDATA_ONLY_NATIVE=0
+NETDATA_ONLY_STATIC=0
+
+NETDATA_DISABLE_TELEMETRY="${DO_NOT_TRACK:-0}"
+NETDATA_TARBALL_BASEURL="${NETDATA_TARBALL_BASEURL:-https://storage.googleapis.com/netdata-nightlies}"
+
+if [ ! -t 1 ]; then
+  INTERACTIVE=0
+else
+  INTERACTIVE=1
+fi
```

Comment:
FWIW, I have had some MacOS users in the past whose terminals where blocking on this test (in an older project not Netdata). I didn't have the time to investigate the reason and I simply reverted my change.

Reply:
I’ve been playing around with this a bit and haven’t been able to trigger any detection issues on macOS. However, the possibility of such issues are why I added an explicit `--interactive` option and kept the existing `--non-interactive` option.